web/app/Models/User.php

```diff
@@ -2,26 +2,62 @@
 
 namespace Wikijump\Models;
 
+use Database\Seeders\UserSeeder;
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Relations\MorphOne;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\Config;
+use Wikijump\Traits\HasSettings;
 
+/**
+ * Class User
+ * @package Wikijump\Models
+ * @mixin Builder
+ */
 class User extends Authenticatable
 {
-    use HasFactory, Notifiable;
+    use HasFactory;
+    use Notifiable;
+    use SoftDeletes;
+    use HasSettings;
+
+    /**
+     * These are service accounts added by the UserSeeder. They're used during
+     * the generation of various pages, threads, and so on. This is their ID.
+     * @see UserSeeder
+     */
+    public const AUTOMATIC_USER = 2;
```

Comment:
My intent was originally just to handle the negative number users since they were the most relevant. I can probably do that as well, but all of the legacy code that is looking for user ID 1 will also end up being replaced with model policies fairly soon.

Reply:
Fair point, it probably isn't worth the time to replace it all.